### PR TITLE
added apeel sciences

### PR DESCRIPTION
**Pull request by code-v1**
## Checklist

- [ x] **I reviewed 2 other [open pull requests](https://github.com/nglgzz/awesome-clean-tech/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- [x ] **I read and followed the [CONTRIBUTING](../CONTRIBUTING.md) guidelines.**
- [ x] **This pull request has a title in the format `Add Company Name`**. Or `Add Company1, Company2 and Company3` if it adds multiple companies.
  - :x: Update README.md
  - :x: Added Orbital Systems
  - :white_check_mark: Add Orbital Systems
  - :white_check_mark: Add Orbital Systems and Bulb
- [ x] **I added entries in alphabetical order.**
- [ x] **The description is simple and doesn't contain unnecessary buzzwords.** It needs to say what the company does, it's not a sales pitch.
  - :x: Disruptive company developing cutting-edge AI-enabled solutions to provide farmers with a decision support system.
  - :white_check_mark: They use satellites to help farmers monitor crops and lower their emissions by reducing their nitrogen usage.

### CODE REVIEW

**Comment by trvrfrd** [comment]
Thanks for contributing!

I would make one stylistic change: avoid repeating the name of the company in the description, e.g. "They develop plant-derived..."

File: README.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ Short description.
 
 ### Vegetation, Agriculture & Food
 
+- [Apeel Sciences](https://apeelsciences.com/) | California, USA | 2012
+
+Apeel Sciences develops plant-derived shelf life extension technology for fresh produce that improves quality and reduces food waste.
```
Comment (aalises) [inline README.md:34]: I would rephrase the description to " _develops a technology that consists of a plant-derived shelf for fresh produce that extends its life improves quality and reduces food waste._ ", for better clarity.
